_posts/2020-06-05-Kim.md

```diff
@@ -0,0 +1,11 @@
+---
+title: A place I'd like to visit when things get better...
+author: Kim Coterwas
+date: 2020-06-05
+---
+
+I'm looking forward to playing air hockey at Dave and Busters with my brother.
+
+![Air Hockey](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F75787206206008293%2F&psig=AOvVaw1-WEXAV0L3rP522aWp0fOZ&ust=1591452544607000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJClwO_s6ukCFQAAAAAdAAAAABAE)
```

Comment:
Image file is a non-valid link

Reply:
probs cause it was from pinterest whoops

Reply:
[Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)